synthea-hiv/generator/hiv_simple/antiretrovirals_started.json

```diff
@@ -0,0 +1 @@
: [{"system": "LOINC", "code": "1250AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "display": "ANTIRETROVIRALS STARTED", "value_set": ""}], "direct_transition": "Terminal", "name": "Tenofovir", "value_code": {"system": "SNOMED-CT", "code": "84795AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "display": "Tenofovir"}}, "Efavirenz": {"type": "Observation", "category": "survey", "unit": "", "codes": [{"system": "LOINC", "code": "1250AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "display": "ANTIRETROVIRALS STARTED", "value_set": ""}], "direct_transition": "Terminal", "name": "Efavirenz", "value_code": {"system": "SNOMED-CT", "code": "75523AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "display": "Efavirenz"}}, "Lamivudine": {"type": "Observation", "category": "survey", "unit": "", "codes": [{"system": "LOINC", "code": "1250AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "display": "ANTIRETROVIRALS STARTED", "value_set": ""}], "direct_transition": "Terminal", "name": "Lamivudine", "value_code": {"system": "SNOMED-CT", "code": "78643AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "display": "Lamivudine"}}, "Abacavir": {"type": "Observation", "category": "survey", "unit": "", "codes": [{"system": "LOINC", "code": "1250AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "display": "ANTIRETROVIRALS STARTED", "value_set": ""}], "direct_transition": "Terminal", "name": "Abacavir", "value_code": {"system": "SNOMED-CT", "code": "70056AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "display": "Abacavir"}}, "Nelfinavir": {"type": "Observation", "category": "survey", "unit": "", "codes": [{"system": "LOINC", "code": "1250AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "display": "ANTIRETROVIRALS STARTED", "value_set": ""}], "direct_transition": "Terminal", "name": "Nelfinavir", "value_code": {"system": "SNOMED-CT", "code": "80487AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "display": "Nelfinavir"}}, "Nevirapine": {"type": "Observation", "category": "survey", "unit": "", "codes": [{"system": "LOINC", "code": "1250AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "display": "ANTIRETROVIRALS STARTED", "value_set": ""}], "direct_transition": "Terminal", "name": "Nevirapine", "value_code": {"system": "SNOMED-CT", "code": "80586AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "display": "Nevirapine"}}, "Lamivudine / zidovudine": {"type": "Observation", "category": "survey", "unit": "", "codes": [{"system": "LOINC", "code": "1250AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "display": "ANTIRETROVIRALS STARTED", "value_set": ""}], "direct_transition": "Terminal", "name": "Lamivudine / zidovudine", "value_code": {"system": "SNOMED-CT", "code": "630AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "display": "Lamivudine / zidovudine"}}, "Lopinavir / ritonavir": {"type": "Observation", "category": "survey", "unit": "", "codes": [{"system": "LOINC", "code": "1250AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "display": "ANTIRETROVIRALS STARTED", "value_set": ""}], "direct_transition": "Terminal", "name": "Lopinavir / ritonavir", "value_code": {"system": "SNOMED-CT", "code": "794AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "display": "Lopinavir / ritonavir"}}}, "gmf_version": 2}
```

Comment:
Can we make this and all other JSON files "pretty", i.e., property indented in multiple lines? Other wise it would be very hard to read the content or track changes.